api/src/datasets.py

```diff
@@ -353,6 +355,24 @@ def put_annotation(payload: MetadataSchema.MetaModel, dataset_id: str):
                     json=feature_payload,
                 )
 
+                feature_response = feature_response.json()
+                concept = feature["primary_ontology_id"]
+
+                if concept != "":
+                    feature_id = feature_response["id"]
+                    concept_payload = {
+                        "curie": concept,
+                        "type": "features",
+                        "object_id": feature_id,
+                        "status": "obj",
+                    }
+
+                    concept_response = requests.post(
```

Comment:
If we don't use `concept_response` (and it doesn't look like we do) we shouldn't capture the return of `requests.post`.
Also, what happens if there is an error creating the concept record as part of this flow? What does that failure mode look like?

Reply:
I removed all the unused response objects from being assigned to variables. I think we are limited in failure modes because the payload is being constructed correctly in the application and the DKG search providing the curie hopefully prevents us from trying to submit incorrectly curie field values. I am not sure that the requests.post would cause any errors at this point.